src/components/DatePicker/DatePicker.jsx

```diff
@@ -40,10 +40,12 @@ export const DatePicker = ({
     timezoneName = null, // seller timezone (e.g. "America/Los_Angeles") to return correct today date
     ...rest
 }) => {
-    const initialValue = variant === variants.single ? value : value.from;
+    const initialValue = variant === variants.single ? value : value?.from;
     const [currentMonth, setCurrentMonth] = useState(initialValue);
     const [startMonth, setStartMonth] = useState(value?.from);
-    const [endMonth, setEndMonth] = useState(value?.to);
+    const [endMonth, setEndMonth] = useState(
+        dayjs(value?.to).isSame(dayjs(value?.from), "month") ? dayjs(value?.from).add(1, "month").toDate() : value?.to,
```

Comment:
if from and to are same month then why are you adding one extra month here for to date ?

Reply:
To have the second datapicker open on the next month, not the same month as the first, if the month is the same
![image](https://github.com/xola/ui-kit/assets/125129104/3a404972-d35a-4840-84ea-c41fec4373ef)